.jenkins/libcxx_tests.Jenkinsfile

```diff
@@ -101,44 +25,27 @@ def ACClibcxxTest(String label, String compiler, String build_type) {
-    }
 
-    registerJenkinsSlaves()
-
-    parallel "libcxx ACC1604 clang-7 Debug" :          { ACClibcxxTest(XENIAL_LABEL, 'clang-7', 'Debug') },
-             "libcxx ACC1604 clang-7 Release" :        { ACClibcxxTest(XENIAL_LABEL, 'clang-7','Release') },
-             "libcxx ACC1604 clang-7 RelWithDebInfo" : { ACClibcxxTest(XENIAL_LABEL, 'clang-7', 'RelWithDebinfo') },
-             "libcxx ACC1604 gcc Debug" :              { ACClibcxxTest(XENIAL_LABEL, 'gcc', 'Debug') },
-             "libcxx ACC1604 gcc Release" :            { ACClibcxxTest(XENIAL_LABEL, 'gcc', 'Release') },
-             "libcxx ACC1604 gcc RelWithDebInfo" :     { ACClibcxxTest(XENIAL_LABEL, 'gcc', 'RelWithDebInfo') },
-             "libcxx ACC1804 clang-7 Debug" :          { ACClibcxxTest(BIONIC_LABEL, 'clang-7', 'Debug') },
-             "libcxx ACC1804 clang-7 Release" :        { ACClibcxxTest(BIONIC_LABEL, 'clang-7', 'Release') },
-             "libcxx ACC1804 clang-7 RelWithDebInfo" : { ACClibcxxTest(BIONIC_LABEL, 'clang-7', 'RelWithDebinfo') },
-             "libcxx ACC1804 gcc Debug" :              { ACClibcxxTest(BIONIC_LABEL, 'gcc', 'Debug') },
-             "libcxx ACC1804 gcc Release" :            { ACClibcxxTest(BIONIC_LABEL, 'gcc', 'Release') },
-             "libcxx ACC1804 gcc RelWithDebInfo" :     { ACClibcxxTest(BIONIC_LABEL, 'gcc', 'RelWithDebinfo') }
+    parallel "libcxx ACC1604 clang-7 Debug" :          { ACClibcxxTest("LIBCXX-1604", 'clang-7', 'Debug') },
+             "libcxx ACC1604 clang-7 Release" :        { ACClibcxxTest("LIBCXX-1604", 'clang-7','Release') },
+             "libcxx ACC1604 clang-7 RelWithDebInfo" : { ACClibcxxTest("LIBCXX-1604", 'clang-7', 'RelWithDebinfo') },
+             "libcxx ACC1604 gcc Debug" :              { ACClibcxxTest("LIBCXX-1604", 'gcc', 'Debug') },
+             "libcxx ACC1604 gcc Release" :            { ACClibcxxTest("LIBCXX-1604", 'gcc', 'Release') },
+             "libcxx ACC1604 gcc RelWithDebInfo" :     { ACClibcxxTest("LIBCXX-1604", 'gcc', 'RelWithDebInfo') },
+             "libcxx ACC1804 clang-7 Debug" :          { ACClibcxxTest("LIBCXX-1804", 'clang-7', 'Debug') },
+             "libcxx ACC1804 clang-7 Release" :        { ACClibcxxTest("LIBCXX-1804", 'clang-7', 'Release') },
+             "libcxx ACC1804 clang-7 RelWithDebInfo" : { ACClibcxxTest("LIBCXX-1804", 'clang-7', 'RelWithDebinfo') },
+             "libcxx ACC1804 gcc Debug" :              { ACClibcxxTest("LIBCXX-1804", 'gcc', 'Debug') },
+             "libcxx ACC1804 gcc Release" :            { ACClibcxxTest("LIBCXX-1804", 'gcc', 'Release') },
+             "libcxx ACC1804 gcc RelWithDebInfo" :     { ACClibcxxTest("LIBCXX-1804", 'gcc', 'RelWithDebinfo') }
 } catch(Exception e) {
     println "Caught global pipeline exception :" + e
     GLOBAL_ERROR = e
     throw e   
 } finally {
-    cleanup()
```

Comment:
We should definitely unregister the slaves as you have done, but I am unsure about deleting the cleanUp functionality. I think it should be modified to no longer delete the resource groups.

Reply:
@BRMcLaren We do not need this function anymore. It was used to delete and unregister the agents spawned by the ACCDeployVM function , which we do not use anymore.
Nodes will be cleaned up by the Azure VM plugin